src/java.base/share/classes/jdk/internal/foreign/abi/BindingSpecializer.java

```diff
@@ -466,9 +468,9 @@ private void doBindings(List<Binding> bindings) {
                 case Copy copy                   -> emitCopyBuffer(copy);
                 case Allocate allocate           -> emitAllocBuffer(allocate);
                 case BoxAddress boxAddress       -> emitBoxAddress(boxAddress);
-                case SegmentBase unused          -> emitSegmentBase();
+                case SegmentBase _               -> emitSegmentBase();
```

Comment:
Seem a bit too far detached from the changes at hand for a drive-by code cleanup?

Reply:
My uncontrollable urges to fix these minor IDE warnings... Please allow these fixes, as I have similar changes in ProxyGenerator as well.